client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue

```diff
@@ -197,7 +197,14 @@ export default {
     },
 
     resetForm () {
-      this.customerContact = emptyCustomer
+      const {title, phoneNumber, eMailAddress, text, visible } = emptyCustomer
+      this.customerContact = {
+        title,
+        phoneNumber,
+        eMailAddress,
+        text,
+        visible
+      }
```

Comment:
what do we want to archive with that?
Isn't it the same as
```suggestion
      this.customerContact = { ...emptyCustomer }
```

Reply:
Yes this does basically the same what I wanted to achieve. Since the empty customer object was not defined in the export default scope I had in mind that vue could not establish the needed reactivity links. Since the spread operator also creates a copy of the emptyCustomer object within the export default scope, your suggestion is leading to less lines of code. This was adjusted here: https://github.com/demos-europe/demosplan-core/pull/2364/commits/6cf2739d3a13f2e971d3802781e78f5e56c6b07e